src/pds_doi_service/core/outputs/osti/osti_validator.py

```diff
@@ -4,29 +4,27 @@
 #  use must be negotiated with the Office of Technology Transfer at the
 #  California Institute of Technology.
 #
-
 """
 =================
 osti_validator.py
 =================
 
 Contains functions for validating the contents of OSTI XML labels.
 """
-
 import tempfile
-from os.path import exists
-
-import xmlschema
 from distutils.util import strtobool
-from pkg_resources import resource_filename
+from os.path import exists
 
+import xmlschema  # type: ignore
 from lxml import etree
-from lxml import isoschematron
-
+from lxml import isoschematron  # type: ignore
 from pds_doi_service.core.entities.doi import DoiStatus
 from pds_doi_service.core.input.exceptions import InputFormatException
 from pds_doi_service.core.outputs.service_validator import DOIServiceValidator
 from pds_doi_service.core.util.general_util import get_logger
+from pkg_resources import resource_filename
+
+# Note that ``lxml`` does have the ``isoschematron`` member, but the typing stub does not, so we ignore:
```

Comment:
Looks like this line needs to be bumped back up to around line 20

Reply:
@MJJoyce looks like `black` re-ordered the imports and lost the comment that went along with it. Is that to be expected?